lang/src/main/java/org/kaazing/k3po/lang/internal/ast/AstConnectNode.java

```diff
@@ -93,4 +94,20 @@ public String getBarrier() {
     public void setBarrier(String barrier) {
         this.barrier = barrier;
     }
+
+    public AstValue getLocation() {
+        return location;
+    }
+
+    public void setLocation(AstValue location) {
```

Comment:
I thought about introducing ``AstLocationExpression`` but ended up reusing ``AstExpressionValue``. That's why I ended up with class hierarchy mentioned up. Any thoughts or suggestions on improving it?